packages/react-components/react-menu/src/components/MenuItem/useMenuItemStyles.ts

```diff
@@ -18,8 +18,22 @@ export const menuItemClassNames: SlotClassNames<MenuItemSlots> = {
 
 const useStyles = makeStyles({
   focusIndicator: createFocusOutlineStyle(),
+  // TODO: this should be extracted to another package
+  resetButton: {
+    boxSizing: 'content-box',
+    backgroundColor: 'inherit',
+    color: 'inherit',
+    fontFamily: 'inherit',
+    fontSize: 'inherit',
+    lineHeight: 'normal',
+    ...shorthands.overflow('visible'),
+    ...shorthands.padding(0),
+    WebkitAppearance: 'button',
+    textAlign: 'unset',
+  },
   root: {
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
+    ...shorthands.borderColor('transparent'),
```

Comment:
should there be a transparent border ? I think most high contrast menus don't have a border separating menuitems actually

Reply:
This should be at `resetButton` actually, thx for the heads up 👍🏻